File: src/Web/WebAngular/src/app/domain/view-models/connector-function-grouped.view-model.ts
import { ConnectorFunctionSummaryViewModel } from "./connector-function-summary.view-model";

export interface ConnectorFunctionGroupedViewModel {
	friendlyName: string;
	name: string;
	connector: string;
	versions: ConnectorFunctionSummaryViewModel[];
	createdBy: string;
	creationDate: string;
	updatedBy: string;
	lastUpdate: string;
}

File: src/Core/Houston.Application/ViewModel/ConnectorFunctionViewModels/ConnectorFunctionGroupedViewModel.cs
namespace Houston.Application.ViewModel.ConnectorFunctionViewModels {
	public class ConnectorFunctionGroupedViewModel {
		public string FriendlyName { get; set; } = null!;
		
		public string Name { get; set; } = null!;

		public string Connector { get; set; } = null!;

		public List<ConnectorFunctionSummaryViewModel> Versions { get; set; } = new();

		public string CreatedBy { get; set; } = null!;

		public DateTime CreationDate { get; set; }

		public string UpdatedBy { get; set; } = null!;

		public DateTime LastUpdate { get; set; }
	}
}
